### ISSUE TITLE
Improve subplot error message

### ISSUE THREAD

**Issue by joa-quim** [GenericMappingTools/gmt#1039]
This happens due to a missing **-c** but the error message is not clear. 
And can't we just skip the need for **-c** for the first pannel of the subplot? Or, in another way, a missing **-c** => **-c1,1**

```
gmt begin lixo

C:\v>gmt subplot begin 1x1  -T"VERY VERY" -R0/5/0/5  -Bwest -Fs7/7

C:\v>echo 1 1 | gmt plot -R0/2/0/2 -Sc0.5 -JX10 -B

C:\v>gmt subplot end
subplot [ERROR]: No subplot information file!
```

**Comment by PaulWessel** [comment]
I agree that it would be nice if a missing -c defaults to -c1,1, and otherwise -c means advance to next panel.  I will see how hard to implement.

**Comment by PaulWessel** [comment]
OK, implemented, please check and hopefully approve #1042.

**Comment by PaulWessel** [comment]
Closed as fixed by #1042.
